README.md

```diff
@@ -578,6 +578,38 @@ module.exports = {
 };
 ```
 
+### insert
+
+Type: `Function`
+Default: `head`
+
+By default, the `extract-css-chunks-plugin` appends styles (`<link>` elements) to `document.head` of the current `window`.
+
+However in some circumstances it might be necessary to have finer control over the append target or even delay `link` elements instertion. For example this is the case when you asynchronously load styles for an application that runs inside of an iframe. In such cases `insert` can be configured to be a function or a custom selector.
+
+If you target an [iframe](https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement) make sure that the parent document has sufficient access rights to reach into the frame document and append elements to it.
+
+#### `insert`
+
+Allows to override default behavior and insert styles at any position.
+
+> ⚠ Do not forget that this code will run in the browser alongside your application. Since not all browsers support latest ECMA features like `let`, `const`, `arrow function expression` and etc we recommend you to use only ECMA 5 features and syntax.
+
+> ⚠ The `insert` function is serialized to string and passed to the plugin. This means that it won't have access to the scope of the webpack configuration module.
```

Comment:
I find this a bit weird and think this could be confusing the users.

Doesn't something simple like `insert: "head" | "body"` fulfill the use case?

Reply:
Initially, I supported both use cases. I can reapply that work?

Reply:
hm, maybe `insert: "head" | "querySelector" | Function`, `querySelector` can be `body`/`#head`/`.class`/etc

Reply:
@sokra I started to work on this feature because I needed to be able to control the insertion bits, specifically inject inside an iframe.

@evilebottnawi suggestions sounds good to me. Can we merge this PR?